mti_nma/steps/avgshape/avgshape.py

```diff
@@ -25,7 +25,7 @@ def __init__(
         self,
         clean_before_run=False,
         direct_upstream_tasks: Optional[List["Step"]] = Shparam,
```

Comment:
Not a part of this PR but this should be a list.
```suggestion
        direct_upstream_tasks: Optional[List["Step"]] = [Shparam],
```